.github/workflows/daily-pr-count.yml

```
@@ -0,0 +1,32 @@
+  schedule:
+    # Run at 10 PM CET (8 PM UTC)
+    - cron: '0 20 * * *'
+  workflow_dispatch:  # Allow manual triggering
+
+jobs:
+  count-prs:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)